Packs/Tenable_io/Integrations/TenableioEventCollector/TenableioEventCollector.py

```diff
@@ -0,0 +1,419 @@
+        return vulnerabilities, False
+    return vulnerabilities, True
+
+
+def generate_export_uuid(client: Client, first_fetch: Optional[datetime], last_run: Dict[str, str | float | None],
+                         severity: List[str]):
+    """
+    Generate a job export uuid in order to fetch vulnerabilities.
+
+    Args:
+        client: Client class object.
+        first_fetch: time to first fetch from.
+        last_run: last run object.
+        severity: severity of the vulnerabilities to include in the export.
+
+    """
+    demisto.info("Getting export uuid for report.")
+    last_fetch = last_run.get('next_fetch_vunl')
+    last_found: float = time.mktime(
+        first_fetch.timetuple()) if not last_fetch and first_fetch else last_fetch  # type: ignore
+    demisto.info(f'{last_found=}')
+    export_uuid = client.get_export_uuid(num_assets=5000, last_found=last_found, severity=severity)
+    set_integration_context({'export_uuid': export_uuid})
+
+    next_run_vuln = time.mktime(datetime.now(tz=timezone.utc).timetuple())
+    new_last_run = {'next_fetch_vunl': next_run_vuln}
+    return new_last_run
+
+
+def run_vulnerabilities_fetch(last_run: dict, first_fetch: Optional[datetime], vuln_fetch_interval: int):
+    """
+
+    Args:
+        last_run: last run object.
+        first_fetch: time to first fetch from.
+        vuln_fetch_interval: vulnerabilities fetch interval.
+
+    Returns: True if fetch vulnerabilities can be run.
+
+    """
+    integration_context = get_integration_context()
+    if not last_run.get('next_fetch_vunl'):
+        time_to_check = time.mktime(first_fetch.timetuple())  # type: ignore
+    else:
+        time_to_check = last_run.get('next_fetch_vunl')  # type: ignore
+    if time.time() - time_to_check > vuln_fetch_interval and not integration_context.get('export_uuid'):
+        return True
+    return False
+
+
+@polling_function('tenable-get-vulnerabilities', requires_polling_arg=False)
+def get_vulnerabilities_command(client: Client, last_found: Optional[float] = None, severity: list = [],
+                                num_assets: int = 5000, export_uuid: str = None):
+    vulnerabilities = []
+    if not export_uuid:
+        export_uuid = client.get_export_uuid(num_assets=num_assets, last_found=last_found, severity=severity)
+
+    status, chunks_available = client.get_export_status(export_uuid=export_uuid)
+    if status == 'FINISHED':
+        for chunk_id in chunks_available:
+            vulnerabilities.extend(client.download_vulnerabilities_chunk(export_uuid=export_uuid, chunk_id=chunk_id))
+        readable_output = tableToMarkdown('Vulnerabilities List:', vulnerabilities,
+                                          removeNull=True,
+                                          headerTransform=string_to_table_header)
+
+        results = CommandResults(outputs_prefix='Tenable.Vulnerabilities',
+                                 outputs_key_field='id',
```

Comment:
unless it's something required for the polling which I'm not aware of:
```suggestion
        results = CommandResults(
```
no need for context path specification for these commands.